### ISSUE TITLE
Cannot create new account

### ISSUE THREAD

**Issue by weslimitless** [nextcloud/mail#609]
### Steps to reproduce
1. Fresh install NextCloud
2. Install Mail Plugin
3. Go to Mail app in Nextcloud
4. Browser JS Exception

### Expected behaviour
- Mail app loads

### Actual behaviour
- Spinner in left hand and right hand panes.
- Exception in browser

### Mail app
Content Security Policy: Directive ‘frame-src’ has been deprecated. Please use directive ‘child-src’ instead.  (unknown)
JQMIGRATE: Migrate is installed, version 1.4.0  core.js:7:542
Starting Mail …  init.js:25
initializing cache…  cache.js:47
TypeError: s is undefined[Learn More]  js.storage.js:127
	_set js.storage.js:127
	_callMethod js.storage.js:438
	set js.storage.js:461
	init cache.js:59
	__WEBPACK_AMD_DEFINE_RESULT__</< app.js:103
	triggerEvents backbone.js:370
	triggerApi backbone.js:356
	eventsApi backbone.js:155
	Events.trigger backbone.js:346
	triggerMethod backbone.marionette.js:167
	start backbone.marionette.js:4438
	__WEBPACK_AMD_DEFINE_RESULT__</< init.js:26
	j http://localhost:8080/core/vendor/core.js:2:26920
	fireWith http://localhost:8080/core/vendor/core.js:2:27738
	ready http://localhost:8080/core/vendor/core.js:2:29530
	I http://localhost:8080/core/vendor/core.js:2:29721

**Mail app version:** (see apps admin page)
0.7.4

**Mailserver or service:** (e.g. Outlook, Yahoo, Gmail, Exchange,...)
None yet, cannot create acct.

**Number of accounts:**

#### Client configuration
**Browser:**
Firefox 56.0.2

**Operating system:**
OSX High Sierra

##### Web server error log

```
Insert your webserver log here
```

##### Server log (data/nextcloud.log)
<details>
```
Insert your server log here
```
</details>

##### Horde IMAP log (data/horde_imap.log)
<details>
```
Insert your horde IMAP log here, see https://github.com/nextcloud/mail#debug-mode
```
</details>

##### Horde SMTP log (data/horde_smtp.log)
<details>
```
Insert your horde SMTP log here, see https://github.com/nextcloud/mail#debug-mode
```
</details>

##### Browser log
<details>
```
Insert your browser log here, this could for example include:

a) The javascript console log
Content Security Policy: Directive ‘frame-src’ has been deprecated. Please use directive ‘child-src’ instead.  (unknown)
JQMIGRATE: Migrate is installed, version 1.4.0  core.js:7:542
Starting Mail …  init.js:25
initializing cache…  cache.js:47
TypeError: s is undefined[Learn More]  js.storage.js:127
	_set js.storage.js:127
	_callMethod js.storage.js:438
	set js.storage.js:461
	init cache.js:59
	__WEBPACK_AMD_DEFINE_RESULT__</< app.js:103
	triggerEvents backbone.js:370
	triggerApi backbone.js:356
	eventsApi backbone.js:155
	Events.trigger backbone.js:346
	triggerMethod backbone.marionette.js:167
	start backbone.marionette.js:4438
	__WEBPACK_AMD_DEFINE_RESULT__</< init.js:26
	j http://localhost:8080/core/vendor/core.js:2:26920
	fireWith http://localhost:8080/core/vendor/core.js:2:27738
	ready http://localhost:8080/core/vendor/core.js:2:29530
	I http://localhost:8080/core/vendor/core.js:2:29721

b) The network log 
c) ...
```
</details>

**Comment by ChristophWurst** [comment]
> Server log (data/nextcloud.log)

Would be great to have those.

**Comment by weslimitless** [comment]
Hi @ChristophWurst.  I checked the logs - there is nothing in them.

Clarification:  I tried mail app with Safari and Chrome and they both work as expected I have been able to create an account, and it is functioning normally. 

My apologies, the bug is not account creation related as I stated in the title.  I couldn't create an account, but apparently that is not the root of the issue.

Even with the account created via Safari, I still cannot view Mail in Firefox. This continues to give a browser exception below.  

![screen shot 2017-11-13 at 7 43 23 am](https://user-images.githubusercontent.com/1740820/32726240-42496260-c846-11e7-8ac7-1db996fd9a3b.png)
